src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -161,131 +181,38 @@ private static ExtensionSettings readExtensionSettings() throws IOException {
         return objectMapper.readValue(file, ExtensionSettings.class);
     }
 
+    /**
+     * This method is call from {@link ExtensionsInitRequestHandler}.
+     * @param extensionTransportService assign value for extensionTransportService
+     */
     void setExtensionTransportService(TransportService extensionTransportService) {
         this.extensionTransportService = extensionTransportService;
     }
 
-    private void setUniqueId(String id) {
+    /**
+     * This method is call from {@link ExtensionsInitRequestHandler}.
```

Comment:
Is it really a call because we are defining a setter here?

Reply:
Ugh, this is where we have some competing desires.  I don't particularly like making the setter public, but it has to be for the init handler (in another package) to set it.  I do think it's fine to indicate where it's intended to be called from, although adding on that it "sets the Unique ID, used in REST requests to uniquely identify this extension" is a good thing.